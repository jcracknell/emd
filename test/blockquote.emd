> line 1
> line 2

Unannounced line:

> line 1
line 2

Multiple paragraphs:

> paragraph 1

> paragraph 2

Nesting:

> level 1
>
>> level 2
>
> level 1
