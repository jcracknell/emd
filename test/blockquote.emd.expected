#! emd.cs.Conversion.StructuralRenderer, emd.cs
(Document
	(Blockquote
		(Paragraph
			(Text "line")
			(Space)
			(Text "1")
			(Space)
			(Text "line")
			(Space)
			(Text "2")
		)	
	)
	(Paragraph
		(Text "Unannounced")
		(Space)
		(Text "line:")
	)	
	(Blockquote
		(Paragraph
			(Text "line")
			(Space)
			(Text "1")
			(Space)
			(Text "line")
			(Space)
			(Text "2")
		)	
	)
	(Paragraph
		(Text "Multiple")
		(Space)
		(Text "paragraphs:")
	)	
	(Blockquote
		(Paragraph
			(Text "paragraph")
			(Space)
			(Text "1")
		)
		(Paragraph
			(Text "paragraph")
			(Space)
			(Text "2")
		)	
	)
	(Paragraph
		(Text "Nesting:")
	)	
	(Blockquote
		(Paragraph
			(Text "level")
			(Space)
			(Text "1")
		)
		(Blockquote
			(Paragraph
				(Text "level")
				(Space)
				(Text "2")
			)
		)	
		(Paragraph
			(Text "level")
			(Space)
			(Text "1")
		)
	)
)
